src/main/java/org/minions/devfund/bruno/WhackAMole.java

```diff
@@ -0,0 +1,233 @@
+package org.minions.devfund.bruno;
+
+import java.util.Random;
+
+/**
+ * This Class in charge to manage the whack a mole game.
+ */
+public class WhackAMole {
+
+    private static final String BLANK_SPACE = " ";
+    private static final int ZERO = 0;
+    private Random random;
+    private String[][] moleGrid;
+    private boolean quitGame;
+    private int score;
+    private int molesLeft;
+    private int attemptsLeft;
+
+    /**
+     * Initializes the constructor for WhackAMole game.
+     *
+     * @param numAttempts   Integer attempts number.
+     * @param gridDimension Integer Dimension grid.
+     */
+    public WhackAMole(final int numAttempts, final int gridDimension) {
+        attemptsLeft = numAttempts;
+        moleGrid = new String[gridDimension][gridDimension];
+        random = new Random();
+        quitGame = false;
+        generateGrid();
+    }
+
+    /**
+     * Generates the grid initial.
+     */
+    private void generateGrid() {
+        int gridLength = moleGrid.length;
+        for (int row = 0; row < gridLength; row++) {
+            for (int col = 0; col < gridLength; col++) {
+                moleGrid[row][col] = "o";
```

Comment:
In order to meet the requirements from edx this should be '*'
`For every spot that has recorded a “whacked mole,” print out a W, or * otherwise.`